### ISSUE TITLE
Device suddenly disconnected without popup box

### ISSUE THREAD

**Issue by fakhrul-omar** [ADFTech/XTEC_X1_Public#26]
Tab title text colour properly changed from black to red, but it somehow bypassed dialog box execution. Another theory is device isn't actually disconnected but text colour somehow changed.

**Comment by kslee911** [comment]
@fakhrul-omar 
This is the bug related to the disconnection, where we have to close the connection to reconnects.

**Comment by fakhrul-omar** [comment]
Not quite. Intended behaviour is whenever the device is disconnected config-tool should popup a message dialog box stating the the device is disconnected (plus changing the title text colour). But in this bug/case the popup doesn't appear. As for why the disconnection occurs is handled by another issue(s).

And yeah, since it's disconnected there's no other way to reconnect other than closing the tab and reopens it.
